docs/user/index.md

```
@@ -0,0 +1,19 @@
+title: User Guide
+slug: /
+description: How to use PROMPT as a user
+sidebar_position: 0
+---
+
+Welcome to the User Guide. This section helps instructors and students use the platform.
+
+See the following pages:
+
+- Getting Started
+	- [Creating a Course](creating_course.md)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 1

(MD007, ul-indent)